libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -533,6 +539,14 @@ class YulUtilFunctions
 	/// signature: (array, newLen)
 	std::string resizeDynamicByteArrayFunction(ArrayType const& _type);
 
+	/// @returns the name of a function that cleans up elements of a storage byte array starting from pos
+	/// it will not copy elements in case of transformation to short byte array, and will not change array length
+	/// In case of pos is greater than len, doesn't do anything
+	/// In case of short byte array (< 32 bytes) doesn't do anything
+	/// Cleans up from first unused slot
```

Comment:
In this aspect it is different from the other "clean end" function, right?

Reply:
Maybe to stress this fact, we could add the word "slots" to the name?

Reply:
I added "slots" to the end of the name. Also, removed shortcut from the other function along with assertion that it was called. with regular array.

Reply:
It wasn't clear to me what this meant at first. How about this?
```suggestion
	/// If the first slot to be cleaned up is partially occupied, does not touch it. Cleans up only completely unused slots.
```